spynnaker/pyNN/extra_algorithms/splitter_components/abstract_spynnaker_splitter_delay.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2020-2021 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from six import add_metaclass
+from spinn_utilities.abstract_base import AbstractBase
+from spynnaker.pyNN.models.neural_projections import (
+    DelayAfferentMachineEdge, DelayedMachineEdge)
+
+
+@add_metaclass(AbstractBase)
+class AbstractSpynnakerSplitterDelay(object):
```

Comment:
Needs a documentation comment saying what the point of this class is. Can be very short.

Reply:
done